webapp/app.py

```diff
@@ -987,6 +987,16 @@ def setup_apispec(handlers):
                             "example": "classification-RHSA-2018:0512"
                         }
                     },
+                    "module_list": {
```

Comment:
it's called module_list but it's an object, should by type of "array" instead of "object" to be consistent with other fields

also it'd be better to call it module**s**_list like in other APIs

perhaps change it to following format?
```
"modules_list": [
    {
        "module_name": "postgresql",
        "module_stream": "10",
        "module_version": "820190104140132",
        "module_context": "9edba152",
        "package_list": []
    }
]
```

or

```
"modules_list": [
    {
        "name": "postgresql",
        "stream": "10",
        "version": "820190104140132",
        "context": "9edba152",
        "package_list": []
    }
]
```

because I don't think `postgresql:10:820190104140132:9edba152` is some standard format